.github/workflows/cleanup.yml

```diff
@@ -31,8 +31,31 @@ jobs:
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TOOL_TOKEN }}"
           PROJ_TOOLS=e1e498-tools PROJ_DEV=e1e498-dev bash openshift/scripts/oc_cleanup_tools.sh ${SUFFIX} apply
 
-      - name: Clean up database (OCP4)
+      - name: Clean up api - test (OCP4)
+        shell: bash
+        run: |
+          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TEST_TOKEN }}"
+          PROJ_TEST=e1e498-test PROJ_DEV=e1e498-dev bash openshift/scripts/oc_cleanup_test.sh ${SUFFIX} apply
+
+    name: Delete DB resources associated with PR
+    runs-on: ubuntu-20.04
```

Comment:
We could add a needs: [Delete non-DB resources associated with pr] - this could be nice, as it's going to clean up everything that uses the db first, then the db.

Reply:
Done